server/app/app.test.js

```diff
@@ -188,6 +189,13 @@ describe('Endpoints', () => {
         .send(updatedItem);
         expect(response.statusCode).toBe(500);  
     });
+    it('should change the reserve property from the default false to true', async () => {
+      const response = await request(app)
+      .patch('/item/5')
+      .send(updatedItem);
+      expect(response.statusCode).toBe(200);
+      expect(response.body.message).toBe('Item 5 has been updated');
+    })
```

Comment:
should only change the reserved value

Reply:
Updated to change only the reserve value